internal/backends/qt/qt_window.rs

```diff
@@ -1567,11 +1568,12 @@ impl WindowAdapterSealed for QtWindow {
             }
         };
 
-        cpp! {unsafe [widget_ptr as "QWidget*",  title as "QString", size as "QSize", background as "QBrush", no_frame as "bool"] {
+        cpp! {unsafe [widget_ptr as "QWidget*",  title as "QString", size as "QSize", background as "QBrush", no_frame as "bool", always_on_top as "bool"] {
             if (size != widget_ptr->size()) {
                 widget_ptr->resize(size.expandedTo({1, 1}));
             }
             widget_ptr->setWindowFlag(Qt::FramelessWindowHint, no_frame);
+            widget_ptr->setWindowFlag(Qt::WindowStaysOnTopHint, always_on_top);
```

Comment:
Found in this list:
https://doc.qt.io/qt-6.2/qtwidgets-widgets-windowflags-example.html